### PR TITLE
First initial version of the Group Creation Use Case. Just wanted to …

### LINKED ISSUE #1: [Feature 4] Group creation

### PR DESCRIPTION
**Pull request by soheegoo**
…commit to show you guys what I've done so far. Probably more things to change/add

### CODE REVIEW

**Review by ileuxali — COMMENTED** [review]
cool

**Review by helenaglow — COMMENTED** [review]
Hi Sohee! I think it would be very helpful if instead of making a pull request with all of the code, you made separate pull requests by package. 

This would make it easier for in depth reviews, and merging :)

Thank you! Let me know if you need anything

**Comment by aaryavatsa** [comment]
Yes, following up on what Helena said -- I need a couple of these files immediately, and so those are the only ones I have reviewed and approved. If you could make a pull request for just the entities that would be great.

**Review by aaryavatsa — COMMENTED** [review]
Yes, I've mostly just looked at our shared entity class changes but this looks good.

File: src/main/java/Entities/CurrentUser.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+        if (currentUser == null){
+            currentUser = new CurrentUser();
+        }
+        return currentUser;
+    }
+    public static void setUser(User newUser){
+        user = newUser;
+    }
+    public static User getUser(){
```
Comment (aaryavatsa) [inline src/main/java/Entities/CurrentUser.java:22]: this makes sense to me! nice implementation -- I think you've followed the design pattern exactly!

File: src/main/java/Entities/CurrentUser.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package Entities;
+import Entities.User;
```
Comment (aaryavatsa) [inline src/main/java/Entities/CurrentUser.java:2]: don't know if you have to import Entities.User if this class is already in the Entities package.

**Review by julia80887 — APPROVED** [review]
looks good

**Review by ileuxali — APPROVED** [review]

**Pull request closed without merging**